### DISCUSSION TITLE
Automatically create/take offer when mining fee falls below certain treshold

### DISCUSSION THREAD

**Discussion by f321x** [bisq-network/bisq#5507] [Ideas]
It would be a very useful feature to be able to create a offer, pay fees upfront and have the option to publish the trade transaction only under a certain mining fee treshold. 

I always have to wait and look multiple times on the mining fee till i can (economically) create a trade where i dont have to overpay on fees. Especially at night fees are low but users cant stay up all night to wait till the fee drops.

I think this feature would be really appreciated by many bisq users and would really help people utilizing bisq especially for smaller trades (stacking sats).

**Comment by pazza83** [comment]
This would be a fantastic addition. More so if it could be used to make multiple offers. i.e. line up trades that will be broadcast when the sats v/b fall to a specific amount.

Not sure if the number would be limited by bitcoinj, as it Bisq does not like it when you make multiple unconfirmed offers. If a solution could be found to accommodate for this even better.

**Comment by ghost** [comment]
concept ACK, it's a great fee-saving feature!
